functional-samples/tutorial.custom-cursor/README.md

```diff
@@ -0,0 +1,9 @@
+# Custom Cursor extension
+
+Adds a custom cursor to pages on https://developer.chrome.com. This sample is part of a tutorial on YouTube.
```

Comment:
were you planning on adding a link to the tutorial once its live?

Reply:
Yes, definitely. I considered blocking this PR on it but I think I might just merge now and open a follow-up PR when the video is live.

Reply:
sgtm